Fixed: Unable to properly highlight input text with mouse

git-anurag-hub opened the pull request:
### Summary

This pr solves the issue https://github.com/elastic/eui/issues/4552

I just removed the space taken between the container div and the input tag which is been taken by the outer div by means on padding and replaced it with space of the input so that now anyone highlighting the input will be in input and not select other nodes.

### Checklist

- [ ] Check against **all themes** for compatibility in both light and dark modes
- [ ] Checked in **mobile**
- [ ] Checked in **Chrome**, **Safari**, **Edge**, and **Firefox**
- [ ] Props have proper **autodocs** and **[playground toggles](https://github.com/elastic/eui/blob/master/wiki/documentation-guidelines.md#adding-playground-toggles)**
- [ ] Added **[documentation](https://github.com/elastic/eui/blob/master/wiki/documentation-guidelines.md)**
- [ ] Checked **[Code Sandbox](https://codesandbox.io/)** works for the any docs examples
- [ ] Added or updated **[jest tests](https://github.com/elastic/eui/blob/master/wiki/testing.md)**
- [ ] Checked for **breaking changes** and labeled appropriately
- [ ] Checked for **accessibility** including keyboard-only and screenreader modes
- [ ] A **[changelog](https://github.com/elastic/eui/blob/master/wiki/documentation-guidelines.md#changelog)** entry exists and is marked appropriately

kibanamachine commented:
Since this is a community submitted pull request, a Jenkins build has not been kicked off automatically. Can an Elastic organization member please verify the contents of this patch and then kick off a build manually?

cchaos commented:
@git-anurag-hub Can you please describe in the PR summary **how** you went about this fix. Providing some before/after screengrabs would be helpful too.

git-anurag-hub commented:
Sure @cchaos, I just removed the space taken between the container div and the input tag which is been taken by the outer div by means on padding and replaced it with space of the input so that now anyone highlighting the input will be in input and not select other nodes.

cchaos commented:
Jenkins, test this

kibanamachine commented:
Preview documentation changes for this PR: https://eui.elastic.co/pr_4593/

cchaos reviewed (changes requested):
Unfortunately these changes have ill-effects on the overall sizing of the combobox sizing. The default size, which should be 40px tall is now too short and both the default size and compressed sizes have too little padding on the left and right sides.

git-anurag-hub commented:
Okay @cchaos let me try something else. If you have any suggestions they are welcomed.
